.github/workflows/e2e-test-train-api.yaml

```diff
@@ -0,0 +1,61 @@
+name: E2E Test with train API
+on:
+  - pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  e2e-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes-version: ["v1.28.7"]
```

Comment:
Please can we run this action on the latest supported Kubernetes version (e.g. v1.31) ?

Reply:
Got it! I will update it.